model_analyzer/config/input/config_command.py

```diff
@@ -96,6 +96,11 @@ def set_config_values(self, args):
         else:
             yaml_config = None
 
+        # Remove 'triton_server_path' validator when in 'remote' mode
```

Comment:
I've started to dig into the CLI/config code for a different story and wow is it involved.  This file you are touching is a very generic high level class that isn't intended to have any knowledge of any specific options.  I don't think this is the correct place to make the change.  I'm going to dig around and try to figure out how all this works.

Reply:
Ack...  I'll look into those.  The tricky part is finding a place where `args` and `config` are both in scope.

Reply:
After review of suggestion: The exception gets thrown during `set_value`.  These set_value calls are inside the loop immediately following my addition to config_command.py.  So the removal of the validation needs to be ahead of those `set_value` calls.  Removing the validation at the time of launching triton will be too late.  Still open for suggestions.

Reply:
[slacked offline] The new approach is to, by default, opt-out validation for the field 'triton_server_path' for all launch modes.  For 'local' launch mode, opt-in validation using `binary_path_validator`.

Reply:
'local' launch mode code now calls binary_path_validator() directly, circumventing the config's internal validator.

Reply:
Done